src/lib/stitching/kaws/v2/stitching.ts

```diff
@@ -29,7 +24,7 @@ export const kawsStitchingEnvironmentV2 = (
     }
     extend type MarketingCollection {
       internalID: ID!
-      artworksConnection(${argsToSDL(filterArtworksArgsV2).join(
+      artworksConnection(${argsToSDL(pageableFilterArtworksArgsWithInput).join(
```

Comment:
Basically, we want to make all the arguments that are possible on the underlying delegated `FilterArtworksConnection` type (which includes root arguments, the `input` type, as well as the connection-specific ones) available to places that stitch it in.